src/aui/tabart.cpp

```diff
@@ -221,7 +221,7 @@ void wxAuiGenericTabArt::SetSizingInfo(const wxSize& tab_ctrl_size,
 {
     m_fixedTabWidth = wxWindow::FromDIP(100, NULL);
 
-    int tot_width = (int)tab_ctrl_size.x - GetIndentSize() - 4;
+    int tot_width = (int)tab_ctrl_size.x - GetIndentSize() - wxWindow::FromDIP(4, NULL);
```

Comment:
This is not a remark for this PR per se, but it's wrong to use `NULL` instead of the actual window here, as this won't work for the windows shown on a secondary display using a different DPI.

We really need to find a way to associate a `wxWindow` with the tab art object, i.e. either pass it as argument to `SetSizingInfo()` or add a separate method to just set the window.

Reply:
I tend to agree, but this was done according to the earlier changes by @MaartenBent and there are multiple places where this would need to be updated (with `wnd` dragged through all the calls). I think it's sufficient for this PR to handle it this way and we can deal with these `FromDIP` calls separately.